host-controller/src/main/java/org/jboss/as/domain/controller/transformers/ServerGroupTransformers.java

```diff
@@ -66,10 +73,10 @@ static ChainedTransformationDescriptionBuilder buildTransformerChain() {
                 .addRejectCheck(RejectAttributeChecker.DEFINED, ServerGroupResourceDefinition.SOCKET_BINDING_DEFAULT_INTERFACE)
                 .end();
 
-        //suspend and resume servers are rejected since 1.8 and below
+        //suspend and resume servers are rejected since Kernel 1.8 (EAP 6.4.0 CP07) and below
         DomainServerLifecycleHandlers.registerServerLifeCycleOperationsTransformers(builder20to18);
 
-        // The use of launch-command is rejected since 2.1 and below
+        // The use of launch-command is rejected since Kernel 2.1 (WildFly Core xx, WildFly 8.1.0) and below
```

Comment:
the WildFly Core xx can be dropped as WF Core did not exist yet (it hadn't been split out from main WF)